R/query.R

```diff
@@ -370,6 +370,41 @@ get_seurat <- function(...) {
 #' @importFrom dplyr tbl
 #' @importFrom httr progress
 #' @importFrom cli cli_alert_info
+#' 
+#' @description 
```

Comment:
The `@description` should only be one paragraph. It should be `@details` if longer. https://roxygen2.r-lib.org/articles/rd.html#the-introduction

Reply:
uups